src/setup/stateful.rs

```diff
@@ -0,0 +1,30 @@
+use std::{io, path::PathBuf};
```

Comment:
This whole module could probably be in `node` instead?

Reply:
Done